editor/inspector_dock.h

```diff
@@ -141,6 +143,7 @@ class InspectorDock : public VBoxContainer {
 	void go_back();
 	void edit_resource(const Ref<Resource> &p_resource);
 	void open_resource(const String &p_type);
+	void open_resource(const Vector<String> &p_types);
```

Comment:
I dont quite understand what this is for yet

Reply:
I did it to allow multiple type of resource to be loaded by "Load Shader" menu option. Because `Shader` and `ShaderInclude` are different kind of resources and I wanted to load either of them using `FileDialog`.